google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/CopyJobConfigurationTest.java

```diff
@@ -115,10 +118,33 @@ public void testToPbAndFromPb() {
 
   @Test
   public void testSetProjectId() {
-    CopyJobConfiguration configuration = COPY_JOB_CONFIGURATION_MULTIPLE_TABLES.setProjectId("p");
-    assertEquals("p", configuration.getDestinationTable().getProject());
+    CopyJobConfiguration configuration = COPY_JOB_CONFIGURATION_MULTIPLE_TABLES.setProjectId(TEST_PROJECT_ID);
```

Comment:
From the code formatter:

```suggestion
    CopyJobConfiguration configuration =
        COPY_JOB_CONFIGURATION_MULTIPLE_TABLES.setProjectId(TEST_PROJECT_ID);
```